finance/proposal-calls/2021-proposal/hire_spectroscopy_developer.md

```diff
@@ -0,0 +1,24 @@
+### Hire specreduce / specutils developer
+
+### Project team
+Moritz Guenther (@hamogu) for this strawmen proposal
```

Comment:
I am willing to join as part of the team

Reply:
i am willing as well. i am also involved with `pypeit` development and am keenly interested in getting the various, separately-funded projects aligned and collaborating more closely.

Reply:
```suggestion
Moritz Guenther (@hamogu), Erik Tollerud (@eteq), Tim Pickering (@tepickering) 
```